traits/ctraits.c

```diff
@@ -5107,20 +5114,21 @@ PyDoc_STRVAR(
     "    is modified.\n");
 
 PyDoc_STRVAR(
-    property_doc,
-    "property()\n"
-    "property(get, get_n, set, set_n, validate, validate_n)\n"
+    _get_property_doc,
+    "_get_property()\n"
     "\n"
-    "Get or set property fields for this trait.\n"
+    "Get the property fields for this trait.\n"
     "\n"
-    "When called with no arguments on a property trait, this method returns a\n"
-    "tuple (get, set, validate) of length 3 containing the getter, setter and\n"
-    "validator for this property trait.\n"
+    "This method returns a tuple (get, set, validate) of length 3 containing\n"
+    "the getter, setter and validator for this property trait.\n"
     "\n"
-    "When called with no arguments on a non-property trait, this method\n"
-    "returns *None*.\n"
+    "When called a non-property trait, this method returns *None*.\n");
+
+PyDoc_STRVAR(
+    _set_property_doc,
+    "_set_property(get, get_n, set, set_n, validate, validate_n)\n"
     "\n"
-    "Otherwise, the *property* method expects six arguments, and uses these\n"
+    "The *property* method expects six arguments, and uses these\n"
```

Comment:
Replace `property` with `_set_property` here, or simply replace with "This method expects [...]".